README.md

```diff
@@ -1,393 +1,74 @@
+Our most recent updates make FLARE VM even more open and maintainable to allow the community to easily add and update tools and make them quickly available to everyone. We've worked hard to open source the packages which detail how to install and configure analysis tools. The FLARE VM project now uses automatic testing, updating, and releasing to make updated packages immediately installable. Read on for more details and all the relevant project and documentation links.
 
-* .NET 4.5 [https://www.microsoft.com/en-us/download/details.aspx?id=30653](https://www.microsoft.com/en-us/download/details.aspx?id=30653)
-* WMF 5.1 [https://www.microsoft.com/en-us/download/details.aspx?id=54616](https://www.microsoft.com/en-us/download/details.aspx?id=54616)
-
-## Version 2.0 Updates
-Starting with version `2.0`, FLARE VM has introduced **breaking changes** with previous versions. A fresh installation in a clean Virtual Machine is recommended.
-
-Starting with version 2.0, FLARE VM uses the following environment variables:
-  - `TOOL_LIST_DIR`: The default value is set to *`%PROGRAMDATA%`*`\Microsoft\Windows\Start Menu\Programs\FLARE`.
-  - `TOOL_LIST_SHORTCUT`: The default value is set to *`%USERPROFILE%`*`\Desktop\FLARE.lnk`.
-
-The installer script sets those environment variables automatically. If there are issues during installation, please verify that those environment variables are set correctly.
+See this [blog](insert URL) for more information regarding changes!
 
 Installation
-=======================
-
-Requirements
-------------
-* 60 GB Hard Drive
-  * Additional space needed after VM is downloaded/installed
-* 2 GB RAM
-
-## Windows 10 Installation
-
-### Pre-installation Steps
-* Download one of the VMs from the links provided above
-  * Import the `.ovf` if necessary  
-* Disable Windows Defender by following the procedures outlined at:
-  * [https://www.howtogeek.com/howto/15788/how-to-uninstall-disable-and-remove-windows-defender-also-how-turn-it-off/](https://www.howtogeek.com/howto/15788/how-to-uninstall-disable-and-remove-windows-defender-also-how-turn-it-off/)
-* Additionally make sure to disable the following settings:
-  * Disable real-time protection
-  * Disable cloud-delivered protection
-  * Disable automatic sample submission
-  * Add the directory `C:\` to the exclusion list
-* Take a snapshot of your machine!
-
-### Installations Steps
-* Download and copy [install.ps1](https://github.com/fireeye/flare-vm/raw/master/install.ps1) onto your new VM
-* Open `PowerShell` as an Administrator
-* Unblock the install file by running:
+===
+
+> **Note:** FLARE VM should ONLY be installed on a virtual machine!
+
+* Prepare a Windows 10+ virtual machine
+  * FLARE VM has been tested on [Windows 10 1809 x64](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/)
+  * We recommend a disk capacity of at least 70-80 GB and memory of at least 2 GB
```

Comment:
```suggestion
  * We recommend to avoid usernames that contain a space or other special characters
  * We recommend a disk capacity of at least 70-80 GB and memory of at least 2 GB
```